main.go

```diff
@@ -150,17 +150,35 @@ var (
 		// They are there, so include them.
 		newPerThreadGaugeMetric("decoder", "packet_size_avg", "", "avg_pkt_size"),
 		newPerThreadGaugeMetric("decoder", "packet_size_max", "", "max_pkt_size"),
+		newPerThreadGaugeMetric("decoder", "mac_address_src_max", "", "max_mac_addrs_src"),
+		newPerThreadGaugeMetric("decoder", "mac_address_dst_max", "", "max_mac_addrs_dst"),
 
 		newPerThreadCounterMetric("decoder", "too_many_layers_total", "", "too_many_layers"),
 	}
 
+	// From: .thread.decoder.event.afpacket
+	// New in 8.0.0: d78f2c9a4e2b59f44daeddff098915084493d08d
+	perThreadDecoderEventAFPacketMetrics = []metricInfo{
+		newPerThreadCounterMetric("decoder_event", "afpacket_trunc_pkt", "", "trunc_pkt").Optional(),
+	}
+
 	// From .thread.flow
 	perThreadFlowMetrics = []metricInfo{
+		// New in 7.0.0: b0993d6fd8ffb6abafdea9324cd0166106af1851
+		newPerThreadCounterMetric("flow", "flow_total", "", "total").Optional(),
```

Comment:
```suggestion
		newPerThreadCounterMetric("flow", "all_total", "", "total").Optional(),
```

Just to prevent `flow_flow_total` and :crossed_fingers: for there never being an "all" metric :-)